### PR TITLE
:sparkles:  adstir native-ad support

### PR DESCRIPTION
**Pull request by shinichiro-takata**
The pull request contains implementation for the adstir native-ad for AMP Ads.

Our CLA: UNITED, Inc.
Our company name: UNITED, Inc.

### CODE REVIEW

File: ads/adstir.md [hunk 1]
```diff
@@ -14,23 +14,37 @@ See the License for the specific language governing permissions and
...
 ## Configuration
 
-For configuration details and to generate your tags, please refer to [your publisher account](https://ad-stir.com/login) or [contact AdStir](https://ad-stir.com/contact).
+For configuration details and to generate your tags, please refer to [your publisher account](https://ad-stir.com/login) or [contact adstir](https://ad-stir.com/contact).
 
 Supported parameters:
 
 - `data-app-id`
 - `data-ad-spot`
+- `data-ad-type`
```
Comment (zhouyx) [inline ads/adstir.md]: Please update the doc, and indicate that `data-app-id` and `data-ad-spot` are required, while the `data-ad-type` is optional. You can refer to our [example doc](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/amp-ad.md#supported-ad-networks).

File: examples/ads.amp.html [hunk 2]
```diff
@@ -440,13 +440,21 @@ <h2>AdSpirit</h2>
...
 
-  <h2>AdStir 320x50 banner</h2>
+  <h2>adstir 320x50 banner</h2>
   <amp-ad width="320" height="50"
       type="adstir"
       data-app-id="MEDIA-343ded3e"
       data-ad-spot="1">
   </amp-ad>
 
+  <h2>adstir native</h2>
```
Comment (zhouyx) [inline examples/ads.amp.html]: The example seems to be a no fill ad when I tested it in Chrome. Is this expected? If so I would suggest either removing, or add a line `<p>Due to ad targeting, the slot might not load ad.</p>`

**Comment by nainar** [comment]
Pinging @shinichiro-takata for next action here.

**Comment by nainar** [comment]
Closing this PR since it hasn't been worked on in >2 months and has merge conflicts. Please reopen if needed.

**Pull request closed without merging**